test/torch/linalg/test_operations.py

```diff
@@ -115,3 +116,20 @@ def test_qr_mpc(hook, workers):
 
     # Check if QR == t
     assert ((Q @ R - t).abs() < 1e-2).all()
+
+
+def test_inv_upper(hook, workers):
+
+    bob = workers["bob"]
+    alice = workers["alice"]
+    crypto_prov = workers["james"]
+
+    torch.manual_seed(0)  # Truncation might not always work so we set the random seed
+    n_cols = 3
+    n_rows = 3
+    R = torch.triu(torch.randn([n_rows, n_cols]))
+    invR = R.inverse()
+
+    R_sh = R.fix_precision(precision_fractional=6).share(bob, alice, crypto_provider=crypto_prov)
+    # invR_sh = DASH._inv_upper(R_sh)
```

Comment:
Just mentioning as a reminder that the end of the test is commented out.

Reply:
Oh thank you, I had completely forgotten about it!

Reply:
*bump*

Reply:
_bump_